contracts/governance/FeeSharingProxy/FeeSharingLogic.sol

```diff
@@ -343,6 +401,53 @@ contract FeeSharingLogic is SafeMath96, IFeeSharingProxy, Ownable, FeeSharingPro
 			"FeeSharingProxy::_getTotalVoluntaryWeightedStake: vested stake exceeds total stake"
 		);
 	}
+
+	/**
+	 * @dev Whitelisting converter address.
+	 *
+	 * @param converterAddress converter address to be whitelisted.
+	 */
+	function addWhitelistedConverterAddress(address converterAddress) external onlyOwner {
+		require(Address.isContract(converterAddress), "Non contract address given");
+		require(converterAddress != address(0), "ERR_ZERO_ADDRESS");
+		require(!isWhitelistedConverter[converterAddress], "WHITELISTED_CONVERTER");
```

Comment:
These verifications overlap.
Pls consider removing duplications.
E.g. no need for `require(converterAddress != address(0), "ERR_ZERO_ADDRESS");` after `Address.isContract(address(0))` and
`		require(Address.isContract(converterAddress), "Non contract address given");` validation can be moved to the function whitelisting converters
```suggestion
		require(!isWhitelistedConverter[converterAddress], "WHITELISTED_CONVERTER");
```

Reply:
This PR has been merged into the PR #385 .
This fix moved to  https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/385/commits/65474a6fdc5ba16dff35acf64a0c5e48b525d930